app/views/api.py

```diff
@@ -898,3 +898,11 @@ def post(self, request, *args, **kwargs):
         except Exception as e:
             print(e)
         return Response()
+
+
+class AwsApiView(APIView):
+    def get(self, request, *args, **kwargs):
+        response = create_presigned_url(
+            settings.AWS_STORAGE_BUCKET_NAME, settings.AWS_SECRET_ACCESS_KEY
+        )
+        return Response(response)
```

Comment:
```suggestion

        return Response(response)
```
Must also return a dict